src/library-authoring/data/api.test.ts

```diff
@@ -118,7 +118,7 @@ describe('library data API', () => {
   it('should add components to unit', async () => {
     const { axiosMock } = initializeMocks();
     const componentId = 'lb:org:lib:html:1';
-    const containerId = 'ltc:org:lib:unit:1';
+    const containerId = 'lcc:org:lib:unit:1';
```

Comment:
nit: shouldn't this be 
```suggestion
    const containerId = 'lct:org:lib:unit:1';
```

Reply:
Ahh, yes! Fixed it.